### DISCUSSION TITLE
Add GUI for Textual Inversion

### DISCUSSION THREAD

**Discussion by dvztimes** [hlky/stable-diffusion-webui#1141] [Feature suggestions]
Pretty self explanatory.  Jut a new UI tab for it.  Something to load the traning images in, fill in the proper fields, and start it running.

I dont think anyone has done this yet.  Once it becomes accessible, it will explode.

**Comment by KobeAmerijckxPT** [comment]
I agree, something like [this dreambooth implementation](https://github.com/XavierXiao/Dreambooth-Stable-Diffusion) would be awesome

**Reply by GitIgnoreMaybe** [reply]
I was just about asking for the same one. You have been just 7 days earlier. 😁
@hlky what do you think about it? Low effort, high impact.
